pkg/scheduler/factory/factory.go

```diff
@@ -239,6 +238,7 @@ type ConfigFactoryArgs struct {
 	PercentageOfNodesToScore       int32
 	BindTimeoutSeconds             int64
 	StopCh                         <-chan struct{}
+	Registry                       framework.Registry
```

Comment:
I may not find the other place using `Registry ` except the initialization of framework . Is it better to just put the `NewRegistry ` into the initialization of framework and remove `Registry ` from `ConfigFactoryArgs `. WDYT? Maybe there is other reason I don't know.

Reply:
The reason for Registry being here is that we want custom scheduler to be able to pass their own registry to the scheduler and have their own set of plugins.